### ISSUE TITLE
Please add a definition for Abnormality of the kinin-kallikrein system (HP:0005559)

### ISSUE THREAD

**Issue by matentzn** [obophenotype/human-phenotype-ontology#11372]
It is not entire certain what this term here encompasses:

https://hpo.jax.org/browse/term/HP:0005559

The fact that it is subsumed under "abnormal circulating protein concentration" suggests that the term despite its general name is supposed to be about abnormal levels of all the proteins associated with the kinin-kallikrein system. Given the position in the hierarchy, this explicitly excludes for example weird interactions between these proteins (with normal levels). 

So either we move the term somewhere else, or we define it to be something like this:

```
Any deviation from the normal concentration of one or more components of the kinin-kallikrein system in the blood circulation.
```

**Comment by pnrobinson** [comment]
@matentzn 
I changed the label to
Abnormal kinin-kallikrein system component activity
and the def to
The activity or concentration of a component of the kinin-kallikrein system is outside the limits of normal.
and the comment:
The kinin-kallikrein system (KKS) is a peptide cascade involved in several physiological processes such as inflammation, coagulation, pain, blood pressure control, and vascular permeability. In the KKS, kinins originate from kininogens through the action of tissue and plasma kallikreins. Some impacts of kinins are induced via activation of B1 and/or B2 receptors and downstream signaling such as nitric oxide.

Probably we need to add a few child terms at some point.
